src/main/java/racingcar/view/OutputView.java

```diff
@@ -0,0 +1,26 @@
+package racingcar.view;
+
+import static racingcar.common.Config.*;
+
+
+public class OutputView {
```

Comment:
```suggestion
public class OutputView {
    private OutputView() {
    }
```
OutputView를 따로 생성하지 않으시는 것 같은데, 그리고 스태틱 메서드로만 이뤄져 있고요. 그러면 InputView 처럼 불필요한 객체 생성을 막기 위해 기본생성자를 private로 생성을 막아두는 게 어떨까요?

Reply:
오 이건 생각을 전혀 못했네요. 감사합니다ㅎㅎ